README.md

```diff
@@ -1,21 +1,58 @@
-# 🇨🇦 🇬🇹 🇰🇷 🇱🇧 Olympick app 🏴󠁧󠁢󠁷󠁬󠁳󠁿 🇵🇰 🇮🇷 🇬🇧 🇨🇴 
 
-Currently there is no way to create a viewing schedule for the Olympics that only includes specific sports.
+# 🇨🇦 🇬🇹 🇰🇷 🇱🇧 🇯🇵 Olympick 🏴󠁧󠁢󠁷󠁬󠁳󠁿 🇵🇰 🇮🇷 🇬🇧 🇨🇴 
 
-This app allow a user to : <br/>
- <li>input their favourite Olympic sport's name and get all the schedules of it.</li><br/>
- <li>input their favourite Olympic sport's name  and get the schedules of specific event of it.</li><br/>
+Currently there is no way to create a viewing schedule for the Olympics that only include specific sports. 
 
-# Tools
+Olympick aims to address this! 
 
-<li> 2020 Olympic Public API : (https://olypi.com/) </li>
-<li> SQL databases</li>
-<li> Python language</li>
-<li> Python Libraries : <strong>Flask</strong>,<strong>Collections</strong>,etc </li>
-<li>Tests : Unit tests and mock tests</li>
+This app allows a user to : <br>
+ <li>Register or log in.</li><br>
+ <li>Input their favourite sports and receive a schedule of all the events happening for those sports.</li><br/>
+ <li>Save chosen events to their personalised schedule.</li>
+<br>
 
 
+## Tools
 
- 
+<li> Python language </li>
+<li> 2020 Olympics Public API : (https://olypi.com/)</li>
+<li> SQL database </li>
+<li> Python Libraries : <strong>Flask</strong>, <strong>Collections</strong>, etc. </li>
+<li> Tests : Unit tests and mock tests </li>
+<br>
+
```

Comment:
We used Flask for creating APIs 
We used mysql connector to connect to SQL database with python
We used HTML files as our templates
we did not use collections, please remove it

Reply:
Done